index.html

```diff
@@ -1,55 +1,65 @@
-            &nbsp;
-            <span class="sixteen">C</span>
-            <span class="one">O</span>
-            <span class="two">N</span>
-            <span class="three">F</span>
-            <span class="four">L</span>
-            <span class="five">I</span>
-            <span class="six">C</span>
-            <span class="seven">T</span>
-            &nbsp;
-            <span class="eight">R</span>
-            <span class="nine">E</span>
-            <span class="ten">S</span>
-            <span class="eleven">O</span>
-            <span class="twelve">L</span>
-            <span class="thirteen">U</span>
-            <span class="fourteen">T</span>
-            <span class="fifteen">I</span>
-            <span class="sixteen">O</span>
-            <span class="one">N</span>
-        </header>
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Pull Requests and Conflict Management</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <div class="container">
+      <header>
+        <span class="one">P</span>
+        <span class="two">U</span>
+        <span class="three">L</span>
+        <span class="four">L</span>
+        &nbsp;
+        <span class="five">R</span>
+        <span class="six">E</span>
+        <span class="seven">Q</span>
+        <span class="eight">U</span>
+        <span class="nine">E</span>
+        <span class="ten">S</span>
+        <span class="eleven">T</span>
+        <span class="twelve">S</span>
+        &nbsp;
+        <span class="thirteen">A</span>
+        <span class="fourteen">N</span>
+        <span class="fifteen">D</span>
+        &nbsp;
+        <span class="sixteen">C</span>
+        <span class="one">O</span>
+        <span class="two">N</span>
+        <span class="three">F</span>
+        <span class="four">L</span>
+        <span class="five">I</span>
+        <span class="six">C</span>
+        <span class="seven">T</span>
+        &nbsp;
+        <span class="eight">R</span>
+        <span class="nine">E</span>
+        <span class="ten">S</span>
+        <span class="eleven">O</span>
+        <span class="twelve">L</span>
+        <span class="thirteen">U</span>
+        <span class="fourteen">T</span>
+        <span class="fifteen">I</span>
+        <span class="sixteen">O</span>
+        <span class="one">N</span>
+      </header>
 
-        <div>
-            <img src="https://ilawyer.com.sg/wp-content/uploads/2020/06/Depositphotos_67457429_l-2015-e1591177273833.jpg" width="600">
-            <p>Before you can begin resolving the conflict, you must wait until both of you are ready. Once you feel that you are calm and able to maintain an honest communication, it is time.</p>
-        </div>
-    </body>
-</html>
\ No newline at end of file
+      <section class="content">
+        <img
+          src="https://www.example.com/images/collaboration.jpg"
+          width="600"
```

Comment:
Change width to 650.
```suggestion
          width="650"
```

Reply:
Changed width to 650